templates/packer-ansible.dockerfile

```
@@ -0,0 +1,16 @@
+FROM ubuntu:18.04
+
+ENV PATH=/root/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common && \
+    apt-add-repository --yes --update ppa:ansible/ansible && \
+    apt-get update && \
+    apt-get install -y ansible unzip wget python3-distutils python3-apt
+
+RUN wget 'https://releases.hashicorp.com/packer/1.5.6/packer_1.5.6_linux_amd64.zip' &&  \
+    unzip 'packer_1.5.6_linux_amd64.zip'
```

```suggestion
RUN wget 'https://releases.hashicorp.com/packer/1.6.6/packer_1.6.6_linux_amd64.zip' &&  \
    unzip 'packer_1.6.6_linux_amd64.zip' -d /usr/local/bin && \
    rm -f packer*.zip
```

I built this Dockerfile locally and it didn't seem to find the `packer` on the `PATH`. You may need to unzip this somewhere else. Also, it looks like the latest `packer` version is `1.6.6`. Will there be any issues if we use that version here instead of `1.5.6`? also might not hurt to remove the `.zip` file after it's unpacked.